### ISSUE TITLE
LineEndConcatenation cop does not check after first line

### ISSUE THREAD

**Issue by mockdeep** [rubocop/rubocop#1043]
I discovered a multi-line string in our app that was like this and didn't trigger a rubocop violation:

``` ruby
@error_msg = 'WARNING: Your class information does not match the ones ' \     
  'needed on this template. They will appear blank on the ' <<     
  'document. Please rename or add fields.'
```

It appears rubocop doesn't scan the additional lines of the string.
